Meadowlark-js/tests/http/local.opensearch.http

```diff
@@ -0,0 +1,93 @@
+#
+# Assumes that you have already run tests in local.33b.http
+#
+
+
+### Authenticate
+# @name client1
+POST http://localhost:3000/local/api/oauth/token
+content-type: application/json
+
+{
+    "grant_type": "client_credentials",
+    "client_id": "meadowlark_key_1",
+    "client_secret": "meadowlark_secret_1"
+}
+
+###
+@authToken1 = {{client1.response.body.$.access_token}}
+
+### 400: improper case on the query term
+GET http://localhost:3000/local/v3.3b/ed-fi/students/?lastSurName=World
+authorization: bearer {{authToken1}}
+
+### 200: actual result (correct case)
+GET http://localhost:3000/local/v3.3b/ed-fi/students/?lastSurname=World
+authorization: bearer {{authToken1}}
+
+### 200: valid limit and offset
+GET http://localhost:3000/local/v3.3b/ed-fi/students?limit=1&offset=2
+authorization: bearer {{authToken1}}
+
+### 400: capitalized limit and offset (Meadowlark is case sensitive, unlike the ODS/API)
+GET http://localhost:3000/local/v3.3b/ed-fi/students?Limit=1&Offset=2
+authorization: bearer {{authToken1}}
+
+### 400: limit without offset
+GET http://localhost:3000/local/v3.3b/ed-fi/students?limit=1
+authorization: bearer {{authToken1}}
+
+### 400: offset without limit
+GET http://localhost:3000/local/v3.3b/ed-fi/students?offset=1
+authorization: bearer {{authToken1}}
+
+### 400: negative limit
+GET http://localhost:3000/local/v3.3b/ed-fi/students?limit=-1&offset=2
+authorization: bearer {{authToken1}}
+
+### 400: negative offset
+GET http://localhost:3000/local/v3.3b/ed-fi/students?limit=1&offset=-2
+authorization: bearer {{authToken1}}
+
+### 400: non-numeric limit
+GET http://localhost:3000/local/v3.3b/ed-fi/students?limit=abc&offset=1
+authorization: bearer {{authToken1}}
+
+### 400: non-numeric offset
+GET http://localhost:3000/local/v3.3b/ed-fi/students?limit=1&offset=abc
+authorization: bearer {{authToken1}}
+
+### 200: multiple query terms
+GET http://localhost:3000/local/v3.3b/ed-fi/students?birthDate=2022-07-28&birthCity=Austin
+authorization: bearer {{authToken1}}
+
+### 400: access a different client's data
+GET http://localhost:3000/local/v3.3b/ed-fi/students?createdBy=another
+authorization: bearer {{authToken1}}
+
+### 400: invalid query term
+GET http://localhost:3000/local/v3.3b/ed-fi/students?asdfasdfa=another
+authorization: bearer {{authToken1}}
+
+### 400: malformed query term
+GET http://localhost:3000/local/v3.3b/ed-fi/students?birth%27City=another&another=another
+authorization: bearer {{authToken1}}
+
```

Comment:
I ran the tests below on the old code. Was "fun" to see the request getting back more results than we would have liked. The final `#` completely comments out the ownership check.